pkg/test/pods.go

```diff
@@ -138,6 +138,15 @@ func UnschedulablePod(options ...PodOptions) *v1.Pod {
 	})...)
 }
 
+// UnschedulablePods returns slice of configurable length of identical test pods with a pending scheduling status condition
+func UnschedulablePods(option PodOptions, num int) []*v1.Pod {
```

Comment:
Can you replace instances of `MakePods` if we are going to use this method? I think `MakePods` does the same thing so would be good to de-dupe.

Reply:
good call

Reply:
done

Had I noticed the local `MakePods` while doing https://github.com/aws/karpenter-core/pull/330 initially I would've just used it, and probably not worried about this change. But I do think this is a teensy bit better and standardizes to all sets of unscheduled pod config usages, so I guess I'm happy I failed to notice that?